src/main/kotlin/lotto/domain/enums/RANK.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.domain.enums
+
+import lotto.domain.Prize
+
+enum class RANK(
+    val sameNumber: Int,
+    val prize: Prize
+) {
+    FIRST(6, Prize(2000000000)),
+    SECOND(5, Prize(1500000)),
+    THIRD(4, Prize(50000)),
+    FOURTH(3, Prize(5000)),
```

Comment:
아래와 같이 3자리 단위로 `_` 구분자를 넣어주면 조금 숫자의 가독성을 높여줄 수 있어요. 😉
```suggestion
    FIRST(6, Prize(2_000_000_000)),
    SECOND(5, Prize(1_500_000)),
    THIRD(4, Prize(50_000)),
    FOURTH(3, Prize(5_000)),
```